src/content/release-notes.mdx

```diff
@@ -5,6 +5,45 @@ sidebar_label: Release notes
 id: release-notes
 ---
 
+## 1.27.0
+
+9 December 2024
+
+This version is compatible with Kubernetes versions 1.27 to 1.30
+
+### Breaking Changes
+
+- Remove code for our deprecated Quickstarts feature <!-- 8842 -->
+
+### New Features
+
+- We now allow the ability for admins of Okteto to [set the minimum supported CLI version for their team](admin/dashboard.mdx#command-line-cli). This will apply for all users who are using CLI 3.2.0 and above <!-- 8829 -->
+- We've published the Okteto Manifest JSON Schema for inline suggestions and validation for creating and editing Okteto manifests within your code editor
+- Added support for overwriting CLI images at the Helm chart level
+- Shipped additional support for the installation of [Okteto in Air-Gapped Environments](self-hosted/manage/air-gapped.mdx) <!-- 8914 -->
+- Configured buildkit probes to only accept requests when buildkit is healthy  <!-- 8871 -->
+- Added support for configuring the Okteto control plane jobs TTL at the helm level  <!-- 8917 -->
+
+### Improvements
+
+- We've reorganized the [Admin Dashboard menu items](admin/dashboard.mdx) into groups for easier navigation  <!-- 8840 -->
+- For Catalog, Cloud Credentials, and Registry Credential items that were created via CRDs, we've added "read-only" tags in the Dashboard to avoid confusion on which items can be edited in the UI <!-- 8886 -->
+- Updated buildkit cacheRatio default value to `0.5`  <!-- 8921 -->
+- The Okteto installer image is no longer needed, binaries for the installer jobs are now installed from the CLI image <!-- 8767 -->
+- Okteto bin image and busybox
```

Comment:
This seems incomplete. These images were merged with the CLI image to simplify the number of docker images to manage. I would prefix it with `[Okteto CLI 3.2.0]`